yt/python/yt/wrapper/default_config.py

```diff
@@ -865,7 +865,6 @@ def _get(d, key):
         if not key.startswith(prefix):
```

Comment:
Do we need this code after the change?

Reply:
Looks like good sanity-check and optimization - all must start with "YT_"